lci_strategic_plugin/include/lci_strategic_plugin/lci_strategic_plugin_config.h

```diff
@@ -0,0 +1,72 @@
+#pragma once
+/*
+ * Copyright (C) 2022 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+#include <string>
+
+namespace lci_strategic_plugin
+{
+//! @brief Struct to store the configuration settings for the LCIStrategicPlugin class
+struct LCIStrategicPluginConfig
+{
+  //! The maximum allowable vehicle deceleration limit in m/s
+  double vehicle_decel_limit = 2.0;
+
+  //! A multiplier to apply to the maximum allowable vehicle deceleration limit so we plan under our capabilities
+  double vehicle_decel_limit_multiplier = 0.75;
+
+  //! The maximum allowable vehicle acceleration limit in m/s
+  double vehicle_accel_limit = 2.0;
+
+  //! A multiplier to apply to the maximum allowable vehicle acceleration limit so we plan under our capabilities
+  double vehicle_accel_limit_multiplier = 0.75;
+
+  //! The length parameter of the participant vehicle used to help calculate the distance before stopping at a red traffic signal
+  double vehicle_length = 5.0;
```

Comment:
Everything related to the vehicle_length in the strategic plugin can be removed. Maneuvers are now operating based on the front bumper of the car and the vehicle_length is accounted for in the plan_delegator. (It will shift maneuver start and end distances by vehicle length)

Reply:
removed all vehicle_length references